db/migrate/20180116210559_add_physical_rack_to_physical_servers.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhysicalRackToPhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_servers, :physical_rack_id, :bigint
```

Comment:
Don't think you need/want to do this in its own migration, adding a column to the new table in the above migration is fine /cc @Fryguy opinions?